src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../Utils/api";
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
```

Comment:
Please add a state for errors and one for a 'loading' indication and please make sure it displays those messages in the appropriate cases.